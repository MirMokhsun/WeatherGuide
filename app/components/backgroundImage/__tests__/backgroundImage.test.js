import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import BackgroundImage from '..';

describe('BackgroundImage', () => {
  test('BackgroundImage snapshot', () => {
    const renderer = new ShallowRenderer();
    const snapshot = renderer.render(<BackgroundImage />);
    expect(snapshot).toMatchSnapshot();
  });
});
