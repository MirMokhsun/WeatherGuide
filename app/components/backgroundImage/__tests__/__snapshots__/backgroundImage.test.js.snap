// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundImage BackgroundImage snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/assets/images/Background2.jpg",
    }
  }
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
